### PR TITLE
jenkins-lts 2.346.2

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
I wanted to try to move this one to OpenJDK 1.17 as well, and to add the changes from another maintainer on `jenkins`done here: https://github.com/Homebrew/homebrew-core/pull/105236/commits/a7a0f9e23347094f1d1c80ebcf35da52f2b4d66a

**Comment by chenrui333** [comment]
> I wanted to try to move this one to OpenJDK 1.17 as well, and to add the changes from another maintainer on `jenkins`done here: [a7a0f9e](https://github.com/Homebrew/homebrew-core/commit/a7a0f9e23347094f1d1c80ebcf35da52f2b4d66a)

that is a good call out.

File: Formula/jenkins-lts.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def caveats
   end
 
   service do
-    run [Formula["openjdk@11"].opt_bin/"java", "-Dmail.smtp.starttls.enable=true", "-jar", opt_libexec/"jenkins.war",
+    run [Formula["openjdk@17"].opt_bin/"java", "-Dmail.smtp.starttls.enable=true", "-jar", opt_libexec/"jenkins.war",
          "--httpListenAddress=127.0.0.1", "--httpPort=8080"]
```
Comment (SMillerDev) [inline Formula/jenkins-lts.rb:34]:
```suggestion
    run [opt_libexec/"jenkins-lts", "--httpListenAddress=127.0.0.1", "--httpPort=8080"]
```

File: Formula/jenkins-lts.rb [hunk 2]
```diff
@@ -14,13 +14,13 @@ class JenkinsLts < Formula
     sha256 cellar: :any_skip_relocation, all: "aa8fecf434630aa80da681b95aa5abd4692c58512cea0b0d6b365e9b19f2c2d5"
   end
 
-  depends_on "openjdk@11"
+  depends_on "openjdk@17"
 
   def install
-    system "#{Formula["openjdk@11"].opt_bin}/jar", "xvf", "jenkins.war"
+    system "#{Formula["openjdk@17"].opt_bin}/jar", "xvf", "jenkins.war"
```
Comment (SMillerDev) [inline Formula/jenkins-lts.rb:20]:
```suggestion
    system Formula["openjdk@17"].opt_bin/"jar", "xvf", "jenkins.war"
```

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
jenkins-lts:
    * Service command does not exist

File: Formula/jenkins-lts.rb [hunk 3]
```diff
@@ -30,8 +30,7 @@ def caveats
   end
 
   service do
-    run [Formula["openjdk@11"].opt_bin/"java", "-Dmail.smtp.starttls.enable=true", "-jar", opt_libexec/"jenkins.war",
-         "--httpListenAddress=127.0.0.1", "--httpPort=8080"]
+    run [opt_libexec/"jenkins-lts", "--httpListenAddress=127.0.0.1", "--httpPort=8080"]
```
Comment (SMillerDev) [inline Formula/jenkins-lts.rb:33]:
```suggestion
    run [opt_bin/"jenkins-lts", "--httpListenAddress=127.0.0.1", "--httpPort=8080"]
```

**Pull request closed without merging**